.travis.yml

```diff
@@ -1,15 +1,44 @@
 language: php
 
+sudo: false
+
 php:
-  - 5.5
   - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
+
+env: SYMFONY_VERSION=2.7.*
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.5
```

Comment:
No, it is still supported. It's not in here because we only run tests against Symfony 2.7.* against it. For better understanding, have a look at https://travis-ci.org/smartive/SmartiveHandlebarsBundle/builds/96084388